systemtests/test_end_to_end.py

```diff
@@ -153,7 +142,7 @@ def test_end_to_end_wish_invalid_rb_number_skipped(self):
         # Create and send json message to ActiveMQ
         data_ready_message = Message(rb_number=self.rb_number,
                                      instrument=self.instrument,
-                                     data=file_location,
+                                     data=str(file_location),
```

Comment:
Is there a case where this is not a string? I briefly looked at `_setup_data_structures` and it seems like it always come out as a str